src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useState } from 'react';
+
+
+export const LoginView = ({onLoggedIn}) => { //onLoggedIn is a prop that will be passed from the MainView component
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const handleSubmit = (event) => { //prevents default action to reload page
+    event.preventDefault();
+
+    const data = {
+      Username: username, 
+      Password: password 
+    }
```

Comment:
This too, needs to be updated to reflect what your backend API expects i.e 
```suggestion
    const data = {
      username: username, 
      password: password 
    }
```
Lower case keys for passwords and username..